.evergreen/run-orchestration.sh

```diff
@@ -5,7 +5,7 @@ set -o errexit  # Exit the script with error if any of the commands fail
 
 AUTH=${AUTH:-noauth}
 SSL=${SSL:-nossl}
-TOPOLOGY=${TOPOLOGY:-server}
+TOPOLOGY=${TOPOLOGY:-standalone}
```

Comment:
For some reason, PHPC uses `standalone` instead of `server`. I can remove this commit, this was part of me trying to get everything to run smoothly and it just stood out. With the additional ticket mentioned above, this file would be removed from the repository in favour of drivers-evergreen-tools anyways.

Reply:
Fine to leave as-is. Seems like something we can address in PHPC-1626.